### PR TITLE
[`integration`] Add Model2Vec as a library on the Hub

### PR DESCRIPTION
**Pull request by tomaarsen**
Hello!

## Pull Request overview
* Add Model2Vec as a library on the Hub
  * Including snippet

## Links
* Repository: https://github.com/MinishLab/model2vec
* Models: https://huggingface.co/models?library=model2vec

## Details
A Sentence Transformers integration is planned. Once released, I'll add a second snippet on how to load the model with Sentence Transformers. 

- Tom Aarsen

### CODE REVIEW

File: packages/tasks/src/model-libraries.ts [hunk 1]
```diff
@@ -406,6 +406,14 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path:"mlc-chat-config.json"`,
 	},
+	model2vec: {
+		prettyLabel: "Model2Vec",
+		repoName: "model2vec",
+		repoUrl: "https://github.com/MinishLab/model2vec",
+		snippets: snippets.model2vec,
+		filter: false,
+		countDownloads: `path_extension:"safetensors"`,
```
Comment (osanseviero) [inline packages/tasks/src/model-libraries.ts:415]: The repos also contain a `config.json` so downloads should be computed by default. Would that work?

**Review by osanseviero — COMMENTED** [review]
Nice :fire:

File: packages/tasks/src/model-libraries.ts (hunk 1 above)
Comment (tomaarsen) [reply packages/tasks/src/model-libraries.ts:415]: Let me double-check whether the `config.json` is actually loaded, or if it's only pushed.

Comment (tomaarsen) [reply packages/tasks/src/model-libraries.ts:415]: Looks like it's indeed used. Let me remove this line for simplicity.

**Review by osanseviero — APPROVED** [review]
LGTM!

**Review by Wauplin — APPROVED** [review]
A clean PR as I like them! Thanks! :hugs:

**Review by pcuenca — APPROVED** [review]

**Pull request merged**